### PR TITLE
Scaffold accessibility documentation for OcButton component

### PR DESCRIPTION
**Pull request by marcus-herrmann**
As suggested by @michaelstingl a first sketch of an accessibility section in a component.

### CODE REVIEW

**Comment by michaelstingl** [comment]
@marcus-herrmann could you post a screenshot? Not all stakeholders want to build it to have a look?

**Comment by marcus-herrmann** [comment]
Bildschirmfoto 2019-10-07 um 16 31 34

File: src/elements/OcButton.vue [hunk 1]
```diff
@@ -22,6 +22,26 @@
...
+ *
+ * 1. The value of the `aria-label` attribute
+ * 2. The text between the opening and closing tag, `<button>This text here</button>`
+ *
+ * When an aria-label attribute exists, its value will override the button text. So in this case, the accessible name would be "foo": `<button aria-label="foo">Bar</button>`, although visual users will see "Bar". This difference between accessible name and visual name is a problem for a certain type of assistive technology ([explainer for the term](https://en.wikipedia.org/wiki/Assistive_technology)), this is why the [WCAG success criterion 2.5.3, "Label in name"](https://www.w3.org/TR/WCAG21/#label-in-name) exists. This difference should be avoided, if it can't, W3C recommends that the accessible name should start with visible label.
+ *
+ * ## Icon-only button
+ *
+ * Every button has to have an accessible name. Can it not be provided by a text between the button tags (for example, because it is an button icon button; see "Upload" example below), the accessible name has to be provided by `aria-label`.
```
Comment (LukasHirt) [inline src/elements/OcButton.vue]: Seems like one extra button found a way in "an button icon button"

Comment (marcus-herrmann) [reply src/elements/OcButton.vue]: Fixed

**Comment by marcus-herrmann** [comment]
Fixed

**Pull request merged**